includes/platforms/configuration/options/allow-urls.mdx

```diff
@@ -1 +1,15 @@
-A list of strings or regex patterns that match error URLs which should exclusively be sent to Sentry. If you use this option, only errors whose entire file URL contains (string) or matches (regex) at least one entry in the list will be sent. As a result, if you add `'foo.com'` to it, it will also match on `https://bar.com/myfile/foo.com`. Keep in mind that this only applies for captured exceptions, not raw message events. By default, all errors are sent.
+An array of strings or regex patterns that match the URLs of scripts where errors have been created.
+Only errors that have been created on these URLs will be sent sent to Sentry.
+If you use this option, only Errors whose top stack frame file URL contains (string) or matches (regex) at least one entry in the array will be sent.
+
+For example, if you add `'foo.com'` to the array, errors created on `https://bar.com/myfile/foo.com` will be captured because URL will be matched with "contains" logic and the last segment of the URL contains `foo.com`.
+
+This matching logic applies for captured exceptions, not raw message events. By default, all errors are sent.
+
+If your scripts are loaded from `cdn.example.com` and your site is `example.com`, you can set `allowUrls` to the follwing to exclusively capture errors being created in scripts in these locations:
```

Comment:
```suggestion
For example, if you add `'foo.com'` to the array, any errors created on `https://bar.com/myfile/foo.com` will be captured because the URL will be matched with "contains" logic (and the last segment of the URL contains `foo.com`).

This matching logic applies to captured exceptions not raw message events. By default, all errors are sent.

If your scripts are loaded from `cdn.example.com` and your site is `example.com`, you can set `allowUrls` to the following to only capture errors being created in scripts in these locations:
```